### ISSUE TITLE
sqlalchemy.exc.InvalidRequestError: A value is required for bind parameter 'id'

### ISSUE THREAD

**Issue by andreasots** [mrphlip/lrrbot#189]
```python-traceback
www.botinteract.APIError: Server error in get_game_id:
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1073, in _execute_context
    context = constructor(dialect, self, conn, *args)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/default.py", line 554, in _init_compiled
    grp, m in enumerate(parameters)]
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/default.py", line 554, in <listcomp>
    grp, m in enumerate(parameters)]
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/sql/compiler.py", line 479, in construct_params
    % bindparam.key)
sqlalchemy.exc.InvalidRequestError: A value is required for bind parameter 'id'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/srv/lrrbot/lrrbot/lrrbot/rpc.py", line 70, in data_received
    response = self.server.functions[request['command']](self.server.lrrbot, request['user'], request['param'])
  File "/srv/lrrbot/lrrbot/lrrbot/rpc.py", line 85, in get_game_id
    return lrrbot.get_game_id()
  File "/srv/lrrbot/lrrbot/common/utils.py", line 100, in decorated_func
    next(x)
  File "/srv/lrrbot/lrrbot/common/utils.py", line 168, in wrapper
    self.lastreturn[params] = yield from func(*args, **kwargs)
  File "/usr/lib/python3.4/asyncio/coroutines.py", line 200, in coro
    res = func(*args, **kw)
  File "/srv/lrrbot/lrrbot/lrrbot/main.py", line 289, in get_game_id
    games.c.name: game_name
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 914, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/sql/elements.py", line 323, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1010, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1078, in _execute_context
    None, None)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1341, in _handle_dbapi_exception
    exc_info
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/util/compat.py", line 200, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/util/compat.py", line 183, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1073, in _execute_context
    context = constructor(dialect, self, conn, *args)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/default.py", line 554, in _init_compiled
    grp, m in enumerate(parameters)]
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/default.py", line 554, in <listcomp>
    grp, m in enumerate(parameters)]
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/sql/compiler.py", line 479, in construct_params
    % bindparam.key)
sqlalchemy.exc.StatementError: (sqlalchemy.exc.InvalidRequestError) A value is required for bind parameter 'id' [SQL: 'INSERT INTO games (id, name) VALUES (%(id)s, %(name)s)'] [parameters: [{Column('id', INTEGER(), table=<games>, primary_key=True, nullable=False, server_default=DefaultClause(<sqlalchemy.sql.elements.TextClause object at 0x7f1995293780>, for_update=False)): 417865, Column('name', TEXT(), table=<games>, nullable=False): 'Roundabout'}]]
```

**Comment by andreasots** [comment]
```python-traceback
www.botinteract.APIError: Server error in get_game_id:
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1073, in _execute_context
    context = constructor(dialect, self, conn, *args)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/default.py", line 554, in _init_compiled
    grp, m in enumerate(parameters)]
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/default.py", line 554, in <listcomp>
    grp, m in enumerate(parameters)]
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/sql/compiler.py", line 479, in construct_params
    % bindparam.key)
sqlalchemy.exc.InvalidRequestError: A value is required for bind parameter 'name'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/srv/lrrbot/lrrbot/lrrbot/rpc.py", line 70, in data_received
    response = self.server.functions[request['command']](self.server.lrrbot, request['user'], request['param'])
  File "/srv/lrrbot/lrrbot/lrrbot/rpc.py", line 85, in get_game_id
    return lrrbot.get_game_id()
  File "/srv/lrrbot/lrrbot/common/utils.py", line 100, in decorated_func
    next(x)
  File "/srv/lrrbot/lrrbot/common/utils.py", line 168, in wrapper
    self.lastreturn[params] = yield from func(*args, **kwargs)
  File "/usr/lib/python3.4/asyncio/coroutines.py", line 200, in coro
    res = func(*args, **kw)
  File "/srv/lrrbot/lrrbot/lrrbot/main.py", line 289, in get_game_id
    games.c.name: game_name
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 914, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/sql/elements.py", line 323, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1010, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1078, in _execute_context
    None, None)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1341, in _handle_dbapi_exception
    exc_info
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/util/compat.py", line 200, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/util/compat.py", line 183, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/base.py", line 1073, in _execute_context
    context = constructor(dialect, self, conn, *args)
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/default.py", line 554, in _init_compiled
    grp, m in enumerate(parameters)]
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/engine/default.py", line 554, in <listcomp>
    grp, m in enumerate(parameters)]
  File "/usr/local/lib/python3.4/dist-packages/sqlalchemy/sql/compiler.py", line 479, in construct_params
    % bindparam.key)
sqlalchemy.exc.StatementError: (sqlalchemy.exc.InvalidRequestError) A value is required for bind parameter 'name' [SQL: 'INSERT INTO games (id, name) VALUES (%(id)s, %(name)s)'] [par
ameters: [{Column('id', INTEGER(), table=<games>, primary_key=True, nullable=False, server_default=DefaultClause(<sqlalchemy.sql.elements.TextClause object at 0x7f1995293780>, for_up
date=False)): 417865, Column('name', TEXT(), table=<games>, nullable=False): 'Roundabout'}]]
```

**Comment by andreasots** [comment]
Hopefully fixed in 564d2d794d683744a7b339bf47aeb60eea1b1421.
